windows/tray_manager_plugin.cpp

```diff
@@ -341,6 +344,46 @@ void TrayManagerPlugin::GetBounds(
   result->Success(flutter::EncodableValue(resultMap));
 }
 
+void TrayManagerPlugin::GetScreenBounds(
+    const flutter::MethodCall<flutter::EncodableValue>& method_call,
+    std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+  const flutter::EncodableMap& args =
+      std::get<flutter::EncodableMap>(*method_call.arguments());
+
+  if (!tray_icon_setted) {
+    result->Success();
+    return;
+  }
+
+  double devicePixelRatio =
+      std::get<double>(args.at(flutter::EncodableValue("devicePixelRatio")));
+
+  flutter::EncodableMap resultMap = flutter::EncodableMap();
+
+  HWND iconWindow = niif.hWnd;
```

Comment:
@rafaelortizzableh This should essentially return the same window handle that the tray icon uses. For that window instance, we're using `MonitorFromWindow` to get the screen of the tray icon.